### ISSUE TITLE
Comments sometimes doubled on docs

### ISSUE THREAD

**Issue by dmcassel** [marklogic/RunDMC#654]
Sometimes the comments section is duplicated on docs.marklogic.com. It's not consistent. When it happens, the second group of comments is formatted normally; the first has very little formatting.

**Comment by sammefford** [comment]
Here's what I see on http://docs.marklogic.com/search:search this morning
![capture](https://cloud.githubusercontent.com/assets/298950/22072473/b54cbd5c-dd5f-11e6-8afe-4842114699dd.PNG)

**Comment by kcoleman-marklogic** [comment]
I wonder if this is somehow browser related. Or (huge stretch) to whether or not you're logged in. I do not see the doubling that Sam gets. I just see the correct stuff below the "3 Comments" header. I'm on Chrome.

**Comment by dmcassel** [comment]
Yeah, I've seen what Sam's showing before, but never been able to reproduce it consistently.
